Products/ZenUI3/browser/resources/js/zenoss/form/graphPanel.js

```diff
@@ -378,10 +379,13 @@
             var p = zenoss.visualization.chart.create(this.graphId, visconfig);
             p.then(function(chart){
                 chart.afterRender = function(){
+                    var legenddiv = chart.$div.find(".nv-legend").length;
```

Comment:
This `afterRender` callback is doing a lot of DOM manipulation which is thrashing the DOM when there are many graphs on the page. Ext provides a way to suspend layout changes, so that may be the solution here.